packages/node/src/subcommands/forceClean.service.ts

```
@@ -0,0 +1,79 @@
+  ) {}
+  private async getExistingProjectSchema(): Promise<string> {
+    let schema = this.nodeConfig.localMode
+      ? DEFAULT_DB_SCHEMA
+      : this.nodeConfig.dbSchema;
+
+    // Note that sequelize.fetchAllSchemas does not include public schema, we cannot assume that public schema exists so we must make a raw query
+    const schemas = (await this.sequelize
+      .query(`SELECT schema_name FROM information_schema.schemata`, {
+        type: QueryTypes.SELECT,
+      })
+      .then((xs) => xs.map((x: any) => x.schema_name))
+      .catch((err) => {
+        logger.error(`Unable to fetch all schemas: ${err}`);
+        process.exit(1);
+      })) as [string];
+
+    if (!schemas.includes(schema)) {
+      // fallback to subqueries table
+      const subqueryModel = await this.subqueryRepo.findOne({
+        where: { name: this.nodeConfig.subqueryName },
+      });
+      if (subqueryModel) {
+        schema = subqueryModel.dbSchema;
+      } else {
+        schema = undefined;
+      }
+    }
+    return schema;
+  }
+
+  async forceClean(): Promise<void> {
+    const schema = await this.getExistingProjectSchema();
+    try {
+      // drop existing project schema and metadata table
+      await this.sequelize.dropSchema(`"${schema}"`, {
+        logging: false,
+        benchmark: false,
+      });
+
+      // remove schema from subquery table (might not exist)
+      await this.sequelize.query(
+        ` DELETE
+                  FROM public.subqueries
+                  WHERE name = :name`,
+        {
+          replacements: { name: this.nodeConfig.subqueryName },
+          type: QueryTypes.DELETE,
+        },
+      );
+
+      logger.info('force cleaned schema and tables');
+
+      if (fs.existsSync(this.nodeConfig.mmrPath)) {
+        await fs.promises.unlink(this.nodeConfig.mmrPath);
+        logger.info('force cleaned file based mmr');
+      }
+    } catch (err) {
+      logger.error(err, 'failed to force clean');
```

Need to rethrow the error